core-bundle/src/Filesystem/Dbafs/Dbafs.php

```diff
@@ -155,7 +155,7 @@ public function getRecords(string $path, bool $deep = false): \Generator
         foreach ($rows as $row) {
             $itemPath = $this->convertToFilesystemPath($row['path']);
 
-            if (!\array_key_exists($itemPath, $this->records)) {
+            if (!\array_key_exists($itemPath, $this->records) || null === $this->records[$itemPath]) {
```

Comment:
```suggestion
            if (!isset($this->records[$itemPath])) {
```

This should be equivalent, shouldn‘t it?